Frameworks/UIKit/UIFont.mm

```diff
@@ -490,41 +129,21 @@ + (UIFont*)systemFontOfSize:(float)size {
 }
 
 /**
- @Status Interoperable
+ @Status Stub
 */
-+ (void)initialize {
-    if (self == [UIFont class]) {
-        g_fontCache = [NSMutableDictionary new];
-        _fontDataCache = [NSMutableDictionary new];
-        _fontList = [[NSMutableDictionary alloc] initWithContentsOfFile:@"/fonts/fontmap.xml"];
-
-        if (!_fontList) {
-            _fontList = [NSMutableDictionary new];
-        }
-
-        _fontInstance = CFDictionaryCreateMutable(NULL, 128, &kCFTypeDictionaryKeyCallBacks, NULL);
-        _fontSizingInstance = CFDictionaryCreateMutable(NULL, 128, &kCFTypeDictionaryKeyCallBacks, NULL);
-        _fontMemory.user = nullptr;
-        _fontMemory.alloc = FTAllocFunc;
-        _fontMemory.free = FTFreeFunc;
-        _fontMemory.realloc = FTReallocFunc;
-        FT_Error error = FT_New_Library(&_fontMemory, &(_fontLib));
-        if (!error) {
-            FT_Add_Default_Modules(_fontLib);
-        } else {
-            TraceWarning(g_logTag, L"Failed to instantiate FreeType library");
-        }
-    }
++ (UIFont*)systemFontOfSize:(CGFloat)size weight:(CGFloat)weight {
+    UNIMPLEMENTED();
+    return StubReturn();
 }
 
 /**
  @Status Interoperable
 */
-+ (UIFont*)boldSystemFontOfSize:(float)size {
++ (UIFont*)boldSystemFontOfSize:(CGFloat)size {
     // TODO 5785385: Using clumsy fontWithDescriptor to initialize here, so that _descriptor is initialized
     // Clean this up a bit once fontDescriptor gets better support
     UIFontDescriptor* fontDes =
-        [[UIFontDescriptor fontDescriptorWithName:@"Helvetica Bold" size:size] fontDescriptorWithSymbolicTraits:UIFontDescriptorTraitBold];
+        [[UIFontDescriptor fontDescriptorWithName:@"Arial Bold" size:size] fontDescriptorWithSymbolicTraits:UIFontDescriptorTraitBold];
```

Comment:
Bit concerned about that actually - the reason why I switched off of Helvetica was because I queried my list of installed fonts through DWrite, and Helvetica actually wasn't in it? So I thought we should probably pick a different default.

---
In reply to: [78640879](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78640879) [](ancestors = 78640879,78640513,78640452,78640310,78639398,78639264)